src/main/java/com/example/planit/controller/AdminController.java

```diff
@@ -70,29 +74,31 @@ public ResponseEntity<DTOstatus> updateCourse(@RequestBody Course course, @Reque
     @CrossOrigin(origins = "http://localhost:3000")
     @GetMapping(value = "/admin/users")
     public ResponseEntity<DTOstatus> getAllUsers(@RequestParam String sub) {
-        DTOusersResponseToController dtOcoursesResponseToController = adminEngine.getAllUsersFromDB(sub);
+        DTOusersResponseToController dtOusersResponseToController = adminEngine.getAllUsersFromDB(sub);
 
-        return ResponseEntity.status(dtOcoursesResponseToController.getHttpStatus())
-                .body(new DTOusersResponseToClient(dtOcoursesResponseToController.isSucceed(),
-                        dtOcoursesResponseToController.getDetails(), dtOcoursesResponseToController.getUsers()));
+        return ResponseEntity.status(dtOusersResponseToController.getHttpStatus())
+                .body(new DTOusersResponseToClient(dtOusersResponseToController.isSucceed(),
+                        dtOusersResponseToController.getDetails(), dtOusersResponseToController.getUsers()));
     }
 
     @CrossOrigin(origins = "http://localhost:3000")
     @PutMapping(value = "/admin/make-user-admin")
     public ResponseEntity<DTOstatus> makeUserAdmin(@RequestParam String sub, @RequestParam String userSubId) {
-        DTOusersResponseToController dtOcoursesResponseToController = adminEngine.makeUserAdminInDB(sub, userSubId);
+        DTOusersResponseToController dtOusersResponseToController = adminEngine.makeUserAdminInDB(sub, userSubId);
 
-        return ResponseEntity.status(dtOcoursesResponseToController.getHttpStatus())
-                .body(new DTOstatus(dtOcoursesResponseToController.isSucceed(),
-                        dtOcoursesResponseToController.getDetails()));
+        return ResponseEntity.status(dtOusersResponseToController.getHttpStatus())
+                .body(new DTOstatus(dtOusersResponseToController.isSucceed(),
+                        dtOusersResponseToController.getDetails()));
     }
 
     @CrossOrigin(origins = "http://localhost:3000")
     @PutMapping(value = "/admin/update-holidays")
     public ResponseEntity<DTOstatus> updateHolidays(@RequestParam String sub) {
 
-        // for oshri to continue implementation
-        return ResponseEntity.status(HttpStatus.OK).body(new DTOstatus(true, Constants.NO_PROBLEM));
+        DTOholidaysResponseToController dtOcoursesResponseToController = adminEngine.updateHolidays(sub);
+
+        return ResponseEntity.status(dtOcoursesResponseToController.getHttpStatus())
+                .body(new DTOstatus(dtOcoursesResponseToController.isSucceed(), dtOcoursesResponseToController.getDetails()));
```

Comment:
It should be `dtOholidaysResponseToController` and not `dtOcourseResponseToController` in the variable name

Reply:
Fixed.